### PR TITLE
MGMT-8087 - Show service logs failures on PROW spyglass

### PR DESCRIPTION
**Pull request by eliorerz**
/cc @osherdp

### CODE REVIEW

File: discovery-infra/junit_log_parser.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+def get_failure_cases(log_file_name: Path, suite_name: str) -> List[TestCase]:
+    test_cases = list()
+
+    with open(log_file_name) as f:
+        file_data = f.readlines()
+
+    for line in file_data:
+        values = re.match(LOG_FORMAT, line)
+        if not values:
```
[the remark this answers is not in the document]
Comment (eliorerz) [reply discovery-infra/junit_log_parser.py:56]: You meant `if values is None`, right?

**Comment by osherdp** [comment]
/hold
let's just try to store it as "flaky" result, other than that code looks great

**Review by osherdp — APPROVED** [review]

**Review by osherdp — APPROVED** [review]

**Pull request merged**